README.md

```diff
@@ -36,11 +36,11 @@ the data that goes into the query string you need to [execute](http://stackoverf
 - Follow the pending RSPEC tests to get your sense of direction.
 
 ### BONUS
-Now that we got every pokemon we want to get them ready to fight. (Did you really think you and Arel weren't going to have a battle after capturing every pokemon?) But if they battle we need to keep track of their hp (health power).  And the only way to do that is to alter the database.  What would be perfect is a sql query that adds an `hp` column and default value of 60 to every row.  That sql command should be put into a migration file in `db/`.
+Now that we got every pokemon we want to get them ready to fight. (Did you really think you and Arel weren't going to have a battle after capturing every pokemon?) But if they battle we need to keep track of their hp (health power).  And the only way to do that is to alter the database.  What would be perfect is a sql query that adds an `hp` column and default value of 60 to every row.  That sql query should be put into a migration file in `db/`.
 
-Once the `hp` column is set up there should be an instance method called `alter_hp` that will allow us to change a specific pokemon's health to a new hp.  It will need to take a new health power as a parameter.
+Once the `hp` column is set up there should be an Pokemon class method called `alter_hp` that will allow us to change a specific pokemon's health to a new hp.  It will need to take the id of the pokemon, a new health power as well as the database we're dealing with as arguments.
```

Comment:
What's your thinking behind making this a class method? To me, this makes more sense remaining as an instance method since we're altering the health of a specific Pokemon.